__tests__/utils.ts

```diff
@@ -64,11 +64,14 @@ export const cli = async (
   }
 ): Promise<string> => {
   const subprocessPromise = new Promise<string>((resolve, reject) => {
-    const subprocess = spawn(`node ${PATH_TO_BINARY} ${flags.join(' ')}`, {
-      shell: true,
-      cwd,
-      stdio: ['inherit', 'pipe', 'inherit'],
-    })
+    const subprocess = spawn(
+      `node ${PATH_TO_BINARY} --skip-telemetry ${flags.join(' ')}`,
```

Comment:
Shouldn't we mock the http request instead and test it as well? Can be separate task, but this sounds like proper solution. You can use something like https://www.npmjs.com/package/fetch-mock-jest or just mock the `toolbox.post`

Reply:
true, I will mock it and add checks whether proper metrics are sent in our tests

Reply:
Actually, I tried to achieve this but I don't see any way to mock `toolbox` or `axios` or anything helpful in integration tests. `jest.mock('gluegun', () => { http: { ... } })` does not work (it's not actually mocked when the test is run).

My understanding is that in integration tests, we run a binary that is composed before tests are run (and modules can be mocked), so when we try to mock, the toolbox is already composed with the real modules and mocking has no effect.

It could be tested in unit tests, where we could just pass mocked `toolbox` as function argument to `run` in `setup-ci.ts` or something like that. But we don't have unit tests :D 

I created draft for this (https://github.com/orgs/software-mansion/projects/24/views/1?pane=issue&itemId=81910427), but I don't know if there actually is a way to achieve this.

This discussion might be related: https://github.com/infinitered/gluegun/issues/416